### PR TITLE
Function to use `frequency_nominal` in place of `channel` for dimension/coordinate

### PR DESCRIPTION
**Pull request by leewujung**
Addresses #710.

## Tasks
- [x] function prototype
- [x] tests: on Sv and MVBS datasets

### CODE REVIEW

File: echopype/utils/common.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import numpy as np
+import xarray as xr
+
+
+def swap_dims_channel_frequency(ds: xr.Dataset) -> xr.Dataset:
+    """
+    Use frequency_nominal in place of channel to be dataset dimension and coorindate.
```
Comment (emiliom) [inline echopype/utils/common.py:7]:
```suggestion
    Use frequency_nominal in place of channel as dataset dimension and coordinate.
```

**Comment by emiliom** [comment]
The function looks good. Thanks!

**Comment by leewujung** [comment]
@emiliom : I added the tests and think this PR is ready to be merged. Let me know what you think.

In the latest commit I also modified the __init__ to allow using this function like the following:
```python
Sv_swap = ep.utils.swap_dims_channel_frequency(Sv)
```

We should definitely revisit putting this function in `./utils/common.py` and the import pattern when we discuss the processing chain.

**Comment by emiliom** [comment]
In the test battery, I don't see what's gained by testing many types of raw data cases. Ultimately, the function `swap_dims_channel_frequency` operates only on `Sv` (and `MVBS`). The form of the resulting `Sv`, after conversion and calibration, should be identical regardless of the source raw data type; if it's not, the test responsible for catching such errors is `test_calibrate.py`, not this one.

I would pare it down to just one or two raw files.

Also, I don't see a specific test for the case of duplicate frequencies. It could be that one of the test files does have duplicated frequencies, but then `_check_swap` should fail. I'd suggest adding a new boolean param value that indicates if the raw file has duplicated frequencies or not, then using that param value to test for the expected outcome of `swap_dims_channel_frequency`.

I'll try to provide a modified version of the test incorporating my suggestions, in the next couple of days.

**Comment by emiliom** [comment]
I've added a PR (https://github.com/leewujung/echopype/pull/5) to your PR, with the changes to the tests that I discussed above. Take a look.

**Comment by leewujung** [comment]
> The form of the resulting `Sv`, after conversion and calibration, should be identical regardless of the source raw data type; if it's not, the test responsible for catching such errors is `test_calibrate.py`, not this one.

I agree -- it is also true that we should add a test to test for consistency of the `compute_Sv/TS` outputs. We currently only have tests to make sure the calibration routines are run through, and in a few limited cases the calibrated output values. I'll make a new issue for this.

**Review by emiliom — APPROVED** [review]
